### ISSUE TITLE
Build failure on Apple Sillicon

### ISSUE THREAD

**Issue by rbrigden** [RustAudio/coreaudio-sys#47]
Hi! I am trying to build CPAL on apple sillicon and coreaudio-sys fails to build for a reason I can't seem to track down. Wondering if you anyone knows a workaround?

```
error: failed to run custom build command for `coreaudio-sys v0.2.6`

Caused by:
  process didn't exit successfully: `/Users/monomoy/reckon/projects/cpal/target/debug/build/coreaudio-sys-7f64659f5c8c26d9/build-script-build` (exit code: 101)
  --- stdout
  cargo:rerun-if-env-changed=COREAUDIO_SDK_PATH
  cargo:rustc-link-lib=framework=AudioToolbox
  cargo:rustc-link-lib=framework=AudioToolbox
  cargo:rustc-link-lib=framework=CoreAudio
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS

  --- stderr
  thread 'main' panicked at 'libclang error; possible causes include:
  - Invalid flag syntax
  - Unrecognized flags
  - Invalid flag arguments
  - File I/O errors
  - Host vs. target architecture mismatch
  If you encounter an error missing from this list, please file an issue or a PR!', /Users/monomoy/.cargo/registry/src/github.com-1ecc6299db9ec823/bindgen-0.53.3/src/ir/context.rs:573:15
  stack backtrace:
     0: rust_begin_unwind
               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca/library/std/src/panicking.rs:495:5
     1: core::panicking::panic_fmt
               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca/library/core/src/panicking.rs:92:14
     2: core::option::expect_failed
               at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca/library/core/src/option.rs:1260:5
     3: core::option::Option<T>::expect
               at /Users/monomoy/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/src/rust/library/core/src/option.rs:349:21
     4: bindgen::ir::context::BindgenContext::new
               at /Users/monomoy/.cargo/registry/src/github.com-1ecc6299db9ec823/bindgen-0.53.3/src/ir/context.rs:567:13
     5: bindgen::Bindings::generate
               at /Users/monomoy/.cargo/registry/src/github.com-1ecc6299db9ec823/bindgen-0.53.3/src/lib.rs:2106:27
     6: bindgen::Builder::generate
               at /Users/monomoy/.cargo/registry/src/github.com-1ecc6299db9ec823/bindgen-0.53.3/src/lib.rs:1419:9
     7: build_script_build::build
               at ./build.rs:124:20
     8: build_script_build::main
               at ./build.rs:139:5
     9: core::ops::function::FnOnce::call_once
               at /Users/monomoy/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
  note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by frewsxcv** [comment]
You'll need to `cargo update`. This was fixed in 0.2.8 https://github.com/RustAudio/coreaudio-sys/pull/45

**Comment by rbrigden** [comment]
Ah thanks, works now :)
